src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1771,9 +1772,11 @@ private IUsingOperation CreateBoundUsingStatementOperation(BoundUsingStatement b
             ImmutableArray<ILocalSymbol> locals = boundUsingStatement.Locals.GetPublicSymbols();
             bool isAsynchronous = boundUsingStatement.AwaitOpt != null;
             IMethodSymbol? disposeMethod = boundUsingStatement.DisposeMethodOpt.GetPublicSymbol();
+            var receiver = boundUsingStatement.ExpressionOpt ?? boundUsingStatement.DeclarationsOpt?.LocalDeclarations[0].DeclaredTypeOpt;
```

Comment:
>DeclaredTypeOpt [](start = 122, length = 15)

This looks fragile, can this be null. Perhaps it would be more robust to pull the type from the local symbol?